pom.xml

```
@@ -30,12 +30,19 @@
         </sonar.coverage.jacoco.xmlReportPaths>
         <sonar.exclusions>
             **/uz/enterprise/mytex/MytexApplication.java,
-            **/uz/enterprise/mytex/config/**/*,
+            **/uz/enterprise/mytex/configs/**/*,
             **/uz/enterprise/mytex/constant/**/*,
             **/uz/enterprise/mytex/entity/**/*,
             **/uz/enterprise/mytex/repository/**/*,
-            **/uz/enterprise/mytex/exception/**/*,
-            **/uz/enterprise/mytex/common/**/*
+            **/uz/enterprise/mytex/exceptions/**/*,
+            **/uz/enterprise/mytex/common/**/*,
+            **/uz/enterprise/mytex/dto/**/*,
+            **/uz/enterprise/mytex/enums/**/*,
+            **/uz/enterprise/mytex/filters/**/*,
```

```suggestion
            **/uz/enterprise/mytex/filter/**/*,
```
use singular nouns for packages